tests/unit_tests/test_wave_06.py

```diff
@@ -76,16 +76,18 @@ def test_swap_best_by_category():
         their_priority="Decor"
     )
 
-    raise Exception("Complete this test according to comments below.")
-    # *********************************************************************
-    # ****** Complete Assert Portion of this test **********
-    # *********************************************************************
-    # Assertions should check:
-    # - That the results is truthy
-    # - That tai and jesse's inventories are the correct length
-    # - That all the correct items are in tai and jesse's inventories, including the items which were swapped from one vendor to the other
+    # print(result)
 
-@pytest.mark.skip
+
+    assert result
+    assert len(tai.inventory) == 3
+    assert len(jesse.inventory) == 3
+    assert item_f in tai.inventory
+    assert item_c in jesse.inventory
+    assert item_f not in jesse.inventory
+    assert item_c not in tai.inventory 
```

Comment:
Good job! I like that you're resting that `item_f` isn't in `jesse`'s inventory and that `item_c` isn't in `tai`'s. I'd check for all three items in both `tai` and `jesse`'s `inventory`.